kythe/cxx/common/indexing/KytheGraphRecorder.cc

```diff
@@ -83,7 +83,7 @@ static const std::string* kEdgeKindSpellings[] = {
     new std::string("/kythe/edge/ref/id"),
     new std::string("/kythe/edge/ref/writes"),
     new std::string("/kythe/edge/ref/writes/implicit"),
-};
+    new std::string("/kythe/edge/influences")};
```

Comment:
It seems to me that these should be added to the schema doc before we start emitting them?

Reply:
These new edges are experimental and aren't yet ready to be included in the official schema.

Reply:
Sure, but they can be labelled as such in the schema.  It's easier to evaluate the code if there is documentation discussing the intended semantics of the edges, which we typically put in the schema.